packages/patternfly-4/react-table/src/components/Table/Table.md

```diff
@@ -40,125 +40,179 @@ import {
   Table,
   TableHeader,
   TableBody,
-  sortable,
-  SortByDirection,
-  headerCol,
-  TableVariant,
-  expandable,
-  cellWidth,
   textCenter,
 } from '@patternfly/react-table';
 
-class SimpleTable extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      columns: [
-        { title: 'Repositories' },
-        'Branches',
-        { title: 'Pull requests' },
-        'Workspaces',
-        {
-          title: 'Last Commit',
-          transforms: [textCenter],
-          cellTransforms: [textCenter]
-        }
-      ],
-      rows: [
-        ['one', 'two', 'three', 'four', 'five'],
-        [
-          {
-            title: <div>one - 2</div>,
-            props: { title: 'hover title', colSpan: 3 }
-          },
-          'four - 2',
-          'five - 2'
-        ],
-        [
-          'one - 3',
-          'two - 3',
-          'three - 3',
-          'four - 3',
-          {
-            title: 'five - 3 (not centered)',
-            props: { textCenter: false }
-          }
-        ]
-      ]
-    };
-  }
+function SimpleTable() {
+  const cells = [
+    'Repositories',
+    'Branches',
+    'Pull requests',
+    'Workspaces',
+    'Last Commit'
+  ];
+  const rows = [
+    ['Foo', 2, 0, 6, 1533635470],
+    ['Bar', 1, 11, 2, 1564566670],
+    ['Baz', 6, 4, 99, 1565167870],    
+  ];
+
+  return (
+    <Table caption="Simple Table" cells={cells} rows={rows}>
+      <TableHeader />
+      <TableBody />
+    </Table>
+  );
+}
+```
 
-  render() {
-    const { columns, rows } = this.state;
+## Cell/row options
 
-    return (
-      <Table caption="Simple Table" cells={columns} rows={rows}>
-        <TableHeader />
-        <TableBody />
-      </Table>
-    );
-  }
+```js
+import React from 'react';
+import {
+  Table,
+  TableHeader,
+  TableBody,
+  cellWidth,
+  textCenter,
+} from '@patternfly/react-table';
+
+function SimpleCustomTable() {
+  const cells = [
+    // Equivalent to just passing the string
+    { title: 'Repositories' }, 
+    'Branches',
+    { title: 'Pull requests' },
+    'Workspaces',
+    // Will run the `transform` functions on the header, and the 
+    // `cellTransforms` on the cells. Transformers are used to inject
+    // extra props to the the cell component. In this case we use the
+    // builtin `textCenter` transformer to instruct the cell to apply
+    // the required stylings to center the cell content.
+    {
+      title: 'Last Commit',
+      transforms: [textCenter],
+      cellTransforms: [textCenter]
+    }
+  ];
+  const rows = [
+    ['Foo', 2, 0, 6, 1533635470],
+    [
+      // Cell content can also be defined as a JSX element.
+      // Extra props can be passed to the cell (the `td` element).
+      {
+        title: <div>Bar 👾</div>,
+        props: { title: 'hover title', colSpan: 3 }
+      },
```

Comment:
nice! I think these comments certainly help understanding. That is the goal!

Reply:
Sure!